### PR TITLE
WORK:3387747 SDK for net library changes & recorded tests for Brooklyn Co-existance feature APIs

### PR DESCRIPTION
**Pull request by Nilambari**
This is related to hydra-specs PR:- https://github.com/Azure/hydra-specs-pr/pull/993
Sending code review, will submit only after dependent PR:https://github.com/Azure/hydra-specs-pr/pull/993 is done.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@Nilambari__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (nilamd). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by Nilambari** [comment]
@nupurkot can you Please help do the code review for the specs? Thanks a lot!

File: src/ServiceManagement/Network/Network.Tests/Gateways/SharedKeyV2Tests.cs [hunk 1]
```diff
@@ -0,0 +1,138 @@
+﻿namespace Network.Tests.Gateways
```
Comment (yugangw-msft) [inline src/ServiceManagement/Network/Network.Tests/Gateways/SharedKeyV2Tests.cs]: add license header

**Comment by Nilambari** [comment]
@dud5 Can you Please take a quick look at merged missing changes? Thanks a lot!

File: src/ServiceManagement/Network/Network.Tests/Network.Tests.csproj [hunk 2]
```diff
@@ -533,6 +557,30 @@
     <None Include="SessionRecords\Network.Tests.Routes.SetRouteTests\SetRouteWithNullRouteNameInParameters.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="SessionRecords\Network.Tests.IPForwardingIntegrationTests\SetIPForwardingOnNICSucceeds.json">
```
Comment (dud5) [inline src/ServiceManagement/Network/Network.Tests/Network.Tests.csproj]: It seems like you added the json files (session records) but didn't add the actual tests.

Need to refresh the file NetworkSecurityGroupIntegrationTests.cs,

as well as add some new files for IPForwarding (IPForwardingIntegrationTests.cs) and Routes (GetEffectiveRouteTableTests.cs)

Comment (Nilambari) [inline src/ServiceManagement/Network/Network.Tests/Network.Tests.csproj]: Thanks Ed! Incorporated review comments.

**Comment by Nilambari** [comment]
@markcowl Can you Please help accept this PR?

**Pull request merged**